Tasks/CmdLineV2/package.json

```diff
@@ -18,6 +18,6 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "uuid": "^3.0.1",
-    "azure-pipelines-task-lib": "2.7.0"
+    "azure-pipelines-task-lib": "2.7.1"
```

Comment:
Is CI forcing a bump for changes?

Reply:
Whoops, meant to call out this change - mocking was briefly broken for v2.7.0 because it tried to reference vsts-task-lib (so I get a compile error), this was fixed in 2.7.1 - see this PR for reference - https://github.com/microsoft/azure-pipelines-task-lib/pull/457